test/runner-unix.c

```diff
@@ -372,7 +376,11 @@ void process_cleanup(process_info_t *p) {
 
 /* Move the console cursor one line up and back to the first column. */
 void rewind_cursor(void) {
+#if defined(__MVS__)
+  fprintf(stderr, "\047[2K\r");
+#else
   fprintf(stderr, "\033[2K\r");
```

Comment:
The character encoding on z/OS is "ebcdic" which is quite different from ascii.
In this case, we want to rewind using the "ESC" character which is (in octal) 33 in ascii and 47 in ebcdic.
Here is a cheatsheet for ascii ebcdic translation
[link](http://www.simotime.com/asc2ebc1.htm)